.github/workflows/maven.yml

```diff
@@ -4,15 +4,20 @@ on: [push, pull_request]
 
 jobs:
   build:
-
+    strategy:
+      fail-fast: true
+      matrix:
+        jdk: [8, 11, 14, 16]
```

Comment:
and if you make this this 8, 11
you should see the build pass now and the java 17 support could go to the graalvm ticket then: #582